pandas/core/series.py

```diff
@@ -189,6 +193,18 @@ def __init__(
 
         else:
 
+            if is_empty_data(data) and dtype is None:
+                # gh-17261
+                warnings.warn(
+                    "The default dtype for empty Series will be 'object' instead"
+                    " of 'float64' in the next version. Specify a dtype explicitly"
```

Comment:
It will not actually be "the next version", so I would replace that with a more general "a future version". 
(If we want to be explicit, it should be version 2.0 according to our deprecation policy)

Reply:
> (If we want to be explicit, it should be version 2.0 according to our deprecation policy

umm i think we are. on rolling 3 major versions iirc
we didn’t actually adopt this